tests/foreman/ui/test_dashboard.py

```diff
@@ -442,16 +396,17 @@ def test_positive_task_status(self):
         with Session(self.browser) as session:
             set_context(session, org=org.name)
             self.assertTrue(self.dashboard.validate_task_navigation(
-                'pending', 'state=running&result=pending'))
+                'running', 'state=running&result=pending'))
             self.assertTrue(self.dashboard.validate_task_navigation(
                 'success',
                 'state=stopped&result=success',
                 "Publish content view '{0}'; organization '{1}'".format(
                     content_view.name, org.name)
             ))
             self.assertTrue(self.dashboard.validate_task_navigation(
-                'error', 'state=stopped&result=error'))
+                'warning', 'state=stopped&result=warning'))
```

Comment:
I wasn't able to get tasks 'error' with `error` status on clean machine, but 'warning` was present. Probably it caused by BZ [1] and failed tasks do not obtain `error` status.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1460240